core/services/chainlink/application.go

```diff
@@ -376,6 +377,8 @@ func (app *ChainlinkApplication) Start() error {
 		return err
 	}
 
+	app.LogBroadcaster.SetLatestHeadFromStorage(app.HeadTracker.HighestSeenHead())
```

Comment:
This is racey. If the head tracker has pulled down a new head between `Start()` and now, this head number will be wrong. 
You need to read from the DB _before_ the head tracker starts.

Reply:
ah yes very true!

Reply:
Moved this now to run before the services are started